src/extension/background-script/actions/nostr/getPrivateKey.ts

```diff
@@ -0,0 +1,13 @@
+import { decryptData } from "~/common/lib/crypto";
+
+import state from "../../state";
+
+const getPrivateKey = async () => {
```

Comment:
actually can we avoid this action? 
"no code is better than no code" :) any code can break and if we don't even have a function to return the private key then this can not go wrong. 

If I see this correctly, then we only need this action to prefill the input field in the settings and to warn the user that it will be overwritten.

Reply:
even though we might want to have an "export key" function (but do we need this now?)

Reply:
I generally share your concerns. Anyway I think it would be a good think for users to also be able to copy their private key, also after the initial setup. 

Some clients don't support `window.nostr` and you will need to plug your private key into those clients directly. 🤔

Reply:
hmm. yes. I see. maybe it's OK here. 

how else could we allow people to export it? 🤔

Reply:
an option would be to let the user enter the password (before revealing the key). i guess that's a bit much to add for an experimental feature though.

Reply:
yes, and it would still require the getprivatekey function. 
it's fine. I think we have this for the accounts also.